.env.homa

```diff
@@ -0,0 +1,14 @@
+### testnet urls
+# relayer addr: 0xe3234f433914d4cfcf846491ec5a7831ab9f0bb3
+# relayer substrate addr: 5C82pcyrgW1gCm3jS8yBjo9AEp9HCUXiUkuKA1zWWiACgw1C
+
+KARURA_ETH_RPC=https://eth-rpc-karura-testnet.aca-staging.network
+KARURA_NODE_URL=wss://karura-testnet.aca-staging.network/rpc/karura/ws
+KARURA_PRIVATE_KEY=efb03e3f4fd8b3d7f9b14de6c6fb95044e2321d6bcb9dfe287ba987920254044
```

Comment:
those key don't hold real assets right

Reply:
no it's only test accounts

Reply:
it's address is `0xe3234f433914d4cfcf846491ec5a7831ab9f0bb3`, which is the test account we used for all tutorials/examples